catalog/transformer/catalog-transformer-overlay/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+    <bean id="thumbnailSupplier" class="ddf.catalog.transformer.ThumbnailSupplier"/>
+
+    <bean id="transformer" class="ddf.catalog.transformer.OverlayMetacardTransformer">
+        <argument ref="thumbnailSupplier"/>
```

Comment:
Nit - You can define the arguement bean inline and clean this file up. Actually you can define bean inline to the service and clean it up more.